hacks_mbof/urls.py

```diff
@@ -43,7 +45,20 @@
     url(r'^bower_components/(?P<path>.*)$', serve, {
         'document_root': 'mbofui/bower_components',
     }),
+]
+
+if 'djangosaml2' in settings.INSTALLED_APPS:
+    urlpatterns += (
+        url(r'^accounts/', include('djangosaml2.urls')),
+        url(r'^test/', 'djangosaml2.views.echo_attributes'),
+    )
```

Comment:
Do you mean the "Testing" section of README.rst?  https://github.com/knaperek/djangosaml2/blob/master/README.rst#testing

I don't understand the point you're trying to make.

In that section of the README, they're calling the Django `patterns` constructor with many URL definitions as arguments.  It's not a list or a tuple.

Ocellus isn't calling the `patterns` constructor directly in any of its code.  However, on [line 33 of `hacks_mbof/urls.py`](https://github.com/tl-its-umich-edu/ocellus/pull/51/files/d174f12a753cd6751d6af5658734a921855de261#diff-09323f9e97370530cc00c10868054eebR33), it begins with `urlpatterns = [`, a list.  But on [line 51](https://github.com/tl-its-umich-edu/ocellus/pull/51/files/d174f12a753cd6751d6af5658734a921855de261#diff-09323f9e97370530cc00c10868054eebR51), the code adds a tuple to it, beginning with `urlpatterns += (`.

My suggestion is that it's safer to work with data structures of the same type. Replace the use of those tuples with lists, `urlpatterns += [`.